sbnanaobj/StandardRecord/SREnums.h

```diff
@@ -141,6 +141,9 @@ namespace caf
     kG4UNKNOWN=60
   };// g4_process_
 
+  enum ReweightType_t{
+    kMultiSim, kPMNSigma, kFixed, kDefault
+  };
```

Comment:
Is there a way to avoid duplicating this, to avoid getting out of sync with https://github.com/SBNSoftware/sbnobj/blob/6718dfc3dcf0ec644f20bbdff2527b921c768d4d/sbnobj/Common/SBNEventWeight/EventWeightParameterSet.h#L59? Or avoiding dependencies makes this unfeasible?

Reply:
We can't depend on sbnobj, because it has art stuff, and sbnanaobj/sbnana are art-free. I think it's essentially a necessary evil.
I suppose, say, sbncode which sees both could have a static_assert that checks they correspond.